pet-store-react-hooks/src/Home/Home.test.tsx

```diff
@@ -0,0 +1,286 @@
+import { petKinds, pets, petsList } from '~/Tests/data';
```

Comment:
Lint your code. If you can't enable linting in your IDE, run `npm run lint`.
We'll talk about the rest on Saturday.